### DISCUSSION TITLE
Hook after Login but before redirect

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#9796] [Feature Requests]
A hook before the redirect but after the authentification would be nice. This case would be nice for https://github.com/directus/directus/discussions/8867 which allows to edit the redirect route or to add parameter to the url.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
